src/VisualStudio/Setup/ProvideRoslynBindingRedirection.cs

```diff
@@ -28,17 +28,6 @@ public ProvideRoslynBindingRedirectionAttribute(string fileName)
                 PublicKeyToken = "31BF3856AD364E35",
                 OldVersionLowerBound = "0.7.0.0",
                 OldVersionUpperBound = "1.2.0.0",
-
```

Comment:
Yes, but the merge commit I'm introducing into master already has 1.1 in it's ancestry. Thus, the stabilization to master merge will just be a fast-forward.